lib/bootleg/ssh.ex

```diff
@@ -45,6 +45,7 @@ defmodule Bootleg.SSH do
     |> List.wrap
     |> Enum.map(&ssh_host_options/1)
     |> SSHKit.context
+    |> SSHKit.env(%{"LANG" => "C", "LC_ALL" => "C"})
```

Comment:
This is making the test suite very unhappy. We should set the locale to utf8 if we are going to set it, as thats what elixir is expecting.